Sources/Public/CalendarView.swift

```diff
@@ -247,7 +247,7 @@ public final class CalendarView: UIView {
     cachedAccessibilityElements = nil
     if let element = focusedAccessibilityElement as? OffScreenCalendarItemAccessibilityElement {
       UIAccessibility.post(
-        notification: .layoutChanged,
+        notification: .screenChanged,
```

Comment:
Keeping this change from #129. This scenario happens when a user swipes to an off-screen accessibility element, and triggers the calendar to scroll a bit so that the previously-off-screen elements are now centered in the middle of the calendar's bounds. Since this triggers a significant visual update, `screenChanged` seems appropriate.